src/components/Banner/Banner.less

```diff
@@ -0,0 +1,90 @@
+@banner: lucid-Banner;
+.@{banner} {
+	.box-sizing();
+	border: 1px solid @color-default-borderColor;
+	background-color: @color-default-backgroundColor;
+	font: @font;
+	border-radius: @size-banner-borderRadiaus;
+	padding: 0 @size-banner-spacing-rightLeft;
+	display: flex;
+
+	&.@{banner}-has-close {
+		padding-right: 10px;
+	}
+
+	&.@{banner}-full-width {
+		border-radius: 0;
+	}
+
+	&-has-icon {
+		& > .@{banner}-content {
+			padding: @size-banner-spacing-topBottom + 2px 0;
```

Comment:
Whoops, looks like those changes to CONTRIBUTING aren't in `master` yet. They are landing in #25